.storybook/preview.tsx

```diff
@@ -4,17 +4,12 @@ import "!style-loader!css-loader!sass-loader!../src/styles.scss";
 import React from "react";
 // We also want to add MDX-style documentation here:
 import { addDecorator, addParameters } from "@storybook/react";
-import { DocsContainer, DocsPage } from "@storybook/addon-docs";
 import { withTests } from "@storybook/addon-jest";
 
-import DSProvider from "../src/theme/provider";
+import nyplTheme from "../src/theme";
 import results from "../.jest-test-results.json";
 
 addParameters({
-  docs: {
```

Comment:
Why is this no longer needed?

Reply:
At some point, this setup was needed for the docs page. Newer versions of Storybook included the docs page automatically so it is safe to remove. I didn't know this but it probably wasn't needed for the past few versions. It'll be great to take a look at the Storybook configs and update them -- they're not necessarily old, but an update would be nice.